parser/src/main/java/com/microsoft/gctoolkit/parser/ZGCParser.java

```diff
@@ -323,6 +340,7 @@ private class ZGCForwardReference {
         private ReclaimSummary reclaimed;
         private ReclaimSummary memorySummary;
         private ZGCMetaspaceSummary metaspace;
+        private int memoryScope; // 0: default all heap; 1: young generation; 2: old generation
```

Comment:
I'm thinking enum here?

Reply:
Yes, it's much better than a magic number.